public/data/Chirag-Lal.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Chirag Lal",
+  "type": "personal",
+  "bio": "Front-End Web Developer with a keen interest towards Movies and Music",
+  "avatar": "https://github.com/chiraglal.png",
+  "links": [
+    {
+      "name": "Check Out my Portfolio",
+      "url": "https://github.com/chiraglal",
```

Comment:
I think you should either keep this field or remove the GitHub one.
I recommend adding your portfolio here if you have one. If you don't have one, IMO, it's best not to add the field. :)

What do you think?


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
oh sorry, my bad. I will add my portfolio link.